src/main/org/deidentifier/arx/metric/Metric.java

```diff
@@ -1492,6 +1492,16 @@ public String toString() {
     }
 
     /**
+     * Calculates the score.
```

Comment:
Thanks for pointing this out, I've extended the class MetricConfiguration accordingly